lib/widgets/game_view.dart

```diff
@@ -90,7 +88,14 @@ class _GameViewState extends State<GameView> {
           child: const Text('Choose grid'),
         ),
         const SizedBox(height: 20),
-        const RulesChangingButtons(),
+        RulesChangingButtons(
+          onRulesChanged: (inRowValue, whoWins) {
+            setState(() {
+              inRowValu = inRowValue;
```

Comment:
rename to `inRowValue` and use this.inRowValue to avoid name collision.

Reply:
Done